arborist/__init__.py

```diff
@@ -195,7 +199,15 @@ def study_page(studiesfolder, study):
                            possible_datatypes=possible_datatypes)
 
 
-@app.route(('/folder/<folderpath:studiesfolder>/s/<study>/clinical/create/'))
+@app.route('/folder/<folderpath:studiesfolder>/set_default/', methods=["GET"])
+def set_default_folder(studiesfolder):
+    response = app.make_response(('', 204))
+    response.set_cookie('default_folder', value=studiesfolder)
+    flash('Default folder set.')
```

Comment:
@wardweistra This issue appears in other places as well right, decided not to figure out how to do that properly for now.  I think we should rethink the way these issues pop-up in the first place.